CMakeLists.txt

```diff
@@ -25,7 +25,7 @@
 #
 #########################################################################
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.16)
```

Comment:
Has anyone concerns regarding version 3.16? Is this decision final on the deal.II side?